Database/Corrections/sodNPCFixes.lua

```diff
@@ -86,6 +86,11 @@ function SeasonOfDiscovery:LoadNPCs()
                 [zoneIDs.MULGORE] = {{30.6, 61.2}},
             },
         },
+        [207577] = { -- Lunar Stone
+            [npcKeys.spawns] = {
+                [zoneIDs.MULGORE] = {{35.2, 69.6}},
+            },
```

Comment:
```suggestion
            [npcKeys.spawns] = {
                [zoneIDs.TELDRASSIL] = {{52.8, 79.8}},
                [zoneIDs.MULGORE] = {{35.2, 69.6}},
            },
```

We want to keep the TELDRASSIL location.

I am fine though by reducing these spawns to a single one per zone even thought there are three. Tooltips will show nevertheless 👍🏻

Reply:
Yeah, I think it's best to have them set to a single spawn point as they are so close to each other in both Teldrassil & Mulgore that they just overlap and it looks messy.